app/assets/stylesheets/application.scss

```diff
@@ -552,3 +552,9 @@ html.turbolinks-progress-bar::before {
   display: block;
   float: left;
 }
+
+.ui-autocomplete {
+  height: 200px;
+  overflow-y: scroll;
+  overflow-x: hidden;
```

Comment:
This does nothing as the width is determined by the content and thus never reaches overflow (try creating a very long query and see what happends)